torch/_inductor/autotune_process.py

```diff
@@ -612,10 +618,9 @@ def do_bench(
         else:
             device_idx = device_interface.current_device()
         with device_interface.device(device_idx):  # type: ignore[attr-defined]
-            out = benchmarker.benchmark_gpu(fn)
-            device_interface.synchronize()  # shake out any CUDA errors
```

Comment:
There really is no reason to do a device sync here, both `lazy_benchmark_gpu` and `benchmark_gpu` will sync the device